src/ert/gui/model/snapshot.py

```diff
@@ -300,26 +282,14 @@ def _add_snapshot(self, snapshot: Snapshot, iter_: int) -> None:
         parent = QModelIndex()
         next_iter = len(self.root.children)
         self.beginInsertRows(parent, next_iter, next_iter)
-        self.root.add_child(snapshot_tree, node_id=iter_)
+        self.root.add_child(snapshot_tree)
         self.rowsInserted.emit(parent, snapshot_tree.row(), snapshot_tree.row())
 
     @override
     def columnCount(self, parent: Optional[QModelIndex] = None) -> int:
-        if parent is None:
-            parent = QModelIndex()
-        parent_node = parent.internalPointer()
-        if parent_node is None:
-            count = len(COLUMNS[NodeType.ROOT])
-        else:
-            if isinstance(parent_node, RootNode):
-                count = len(COLUMNS[NodeType.ROOT])
-            elif isinstance(parent_node, IterNode):
-                count = len(COLUMNS[NodeType.ITER])
-            elif isinstance(parent_node, RealNode):
-                count = len(COLUMNS[NodeType.REAL])
-            else:
-                count = len(COLUMNS[NodeType.JOB])
-        return count
+        if parent and isinstance(parent.internalPointer(), RealNode):
+            return JOB_COLUMN_SIZE
+        return 1
```

Comment:
Why do we return 1?

Reply:
The indexing was made to fit arrays/tables, but this isn't a table, so specifying something other than 1 will cause either objects not to be drawn, or returning too many columns.